VERSION

```diff
@@ -0,0 +1,5 @@
+VERSION_MAJOR = 0
```

Comment:
memfault's built-in device info pulls sw version from this if present, using zephyr's application version system